scss/_settings_colors.scss

```diff
@@ -45,11 +45,11 @@ $colors--light-theme--text-muted: #666 !default;
 $colors--light-theme--text-default: #111 !default;
 
 //text-hover: minimum contrast on primary that satisfies contrast checker AA
+$colors--dark-theme--text-default: hsl(0, 0%, 100%) !default;
+$colors--dark-theme--text-disabled: rgba($colors--dark-theme--text-default, 0.55) !default;
+$colors--dark-theme--text-muted: rgba($colors--dark-theme--text-default, 0.55) !default;
+$colors--dark-theme--text-hover: hsl(0, 0%, 56%) !default;
 $colors--dark-theme--background: hsl(0, 0%, 15%) !default;
-$colors--dark-theme--background-highlighted: hsl(0, 0%, 20%) !default;
-$colors--dark-theme--border-default: hsl(0, 0%, 27%) !default;
+$colors--dark-theme--background-highlighted: rgba($colors--dark-theme--text-default, 0.05) !default;
+$colors--dark-theme--border-default: rgba($colors--dark-theme--text-default, 0.05) !default;
```

Comment:
@lyubomir-popov 

Now that we have opacity based values here, I guess we should be consistent and define `$colors--dark-theme--text-hover` and `$colors--dark-theme--border-high-contrast` in the same way - could you suggest the values?

Reply:
Yes, definitely. I'll come up with values.